source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.json

```diff
@@ -92,44 +94,47 @@
           "arguments": {
             "thread": "thread1",
             "operation": {
-                "name": "find",
-                "object": "collection",
-                "arguments": {
-                    "filter": { "$where": "sleep(2000) || true" }
-                },
-                "expectError": {
-                  "isError": true
+              "name": "find",
+              "object": "collection",
+              "arguments": {
+                "filter": {
```

Comment:
I don't understand why this change was applied, but probably it's not a big deal

Reply:
`js-yaml` does not inline JSON objects. By default, keys within an object appear indented on their own line between the opening and closing braces.